tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,207 @@
+# in the beamformer calculation. This can be achieved by setting the parameter
+# ``reg=0.05`` when calculating the spatial filter with
+# :func:`~mne.beamformer.make_lcmv`. This corresponds to loading the diagonal
+# of the covariance matrix with 5% of the sensor power.
+
+
+data_cov = mne.compute_covariance(epochs, tmin=0.01, tmax=0.25,
+                                  method='empirical')
+noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0,
+                                   method='empirical')
+
+data_cov.plot(epochs.info)
+
+###############################################################################
+# The forward model
+# -----------------
+# The forward model is the other important ingredient for the computation of a
+# spatial filter. Here, we will load the forward model from disk, more
+# information on how to create a forward model can be found in this tutorial:
+# :ref:`tut-forward`.
+# Note that beamformers are usually computed in a :class:`volume source space
+# <mne.VolSourceEstimate>`, as a visualization of only the surface activation
+# can misrepresent the data.
+
+# Read forward model
+fwd_fname = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+forward = mne.read_forward_solution(fwd_fname)
+
+
+###############################################################################
+# Compute the spatial filter
+# --------------------------
+# Now we can compute the spatial filter.
+#
+# Different variants of the LCMV beamformer exist. We will compute a
+# unit-noise-gain beamformer, which normalizes the beamformer weights to take
+# care of a depth bias, which is inherent to the forward model solution. To
+# achieve this, we set ``weight_norm='unit-noise-gain'``. Other possibilities
+# to cope with this depth bias are the normalization of the forward model
+# setting the ``depth`` parameter when computing the spatial filter or the
+# contrasting of conditions (then the depth bias will cancel out).
+# This parameter can also be set to ``'nai'`` (Neural Activity Index, see [1]_)
+# which implements
+# a further normalization with the estimated noise. An alternative way to take
+# care of this depth bias is to use the ``depth`` parameter and normalize the
+# forward solution instead of the weights. Note that if you compare conditions,
+# the depth bias will cancel out and it is possible to set both parameters to
+# ``None``.
+#
+# Furthermore, we will optimize the orientation of the sources such that output
+# power is maximized. This is achieved by setting ``pick_ori='max-power'``.
+# This gives us one source estimate per source (i.e., voxel), which is known
+# as a scalar beamformer. It is also possible to compute a vector beamformer,
+# which gives back three estimates per voxel, corresponding to the three
+# directions of the source. This can be achieved by setting
+# ``pick_ori='vector'`` and will yield a :class:`volume vector source estimate
+# <mne.VolVectorSourceEstimate>`.
```

Comment:
```suggestion
# Now we can compute the spatial filter. We'll use "unit noise gain" to deal
# with depth bias, and will also
# optimize the orientation of the sources such that output
# power is maximized. This is achieved by setting ``pick_ori='max-power'``.
# This gives us one source estimate per source (i.e., voxel), which is known
# as a scalar beamformer. It is also possible to compute a vector beamformer,
# which gives back three estimates per voxel, corresponding to the three
# direction components of the source. This can be achieved by setting
# ``pick_ori='vector'`` and will yield a :class:`volume vector source estimate
# <mne.VolVectorSourceEstimate>`.
```